dGame/dComponents/PropertyComponent.h

```diff
@@ -20,14 +20,13 @@ struct PropertyState {
 /**
  * This component is unused and has no functionality
  */
-class PropertyComponent : public Component {
+class PropertyComponent final : public Component {
 public:
-	inline static const eReplicaComponentType ComponentType = eReplicaComponentType::PROPERTY;
+	constexpr static eReplicaComponentType ComponentType = eReplicaComponentType::PROPERTY;
 	explicit PropertyComponent(Entity* parentEntity);
-	~PropertyComponent() override;
-	[[nodiscard]] PropertyState* GetPropertyState() const { return m_PropertyState; };
+	[[nodiscard]] PropertyState* GetPropertyState() const { return m_PropertyState.get(); };
```

Comment:
This class aside from the ComponentType and constructor can be removed.  The component is unused entirely and not worth having stuff still in it.